### ISSUE TITLE
tacl import does not support multiple files (asterix expansion)

### ISSUE THREAD

**Issue by huguesfontenelle** [unioslo/tsd-api-client#47]
## Background

I'm trying to import several files with `*` expansion, but only the first matched file is uploaded.

## Steps to reproduce

```
tacl --import /path/to/folder/*.gz
User > pXX-hugues
Password >
OTP > 123456
```
works but only first file is uploaded.

## Expected behavior

All files should upload :-)

## Workaround

```
find /path/to/folder -name "*.gz" -exec tacl --import {} \;
```
That works, but I need to re-type username, password, and OTP for every file.

## Version

v2.3.0

**Comment by leondutoit** [comment]
Yeah this is a nice feature to have. Will see when we get around to adding it. In the meantime, just create a zip archive.
